bedrock/privacy/views.py

```diff
@@ -40,10 +40,13 @@ def get_legal_doc(self):
 class FirefoxPrivacyDocView(PrivacyDocView):
     def get_legal_doc(self):
         doc = super().get_legal_doc()
-        if len(doc["content"].select(".privacy-header-firefox")) > 0:
-            self.template_name = "privacy/notices/firefox.html"
+        variant = self.request.GET.get("v", None)
+
+        if variant == "product":
+            self.template_name = "privacy/notices/firefox-tos.html"
         else:
-            self.template_name = "privacy/notices/firefox-old-style-notice.html"
```

Comment:
Note for reviewer: I removed this conditional as there does not seem to be any active locales in production that are using this older privacy notice template.